static/src/stylesheets/module/survey/_weekend-reading.scss

```diff
@@ -0,0 +1,177 @@
+.email-signup-main {
+    background-color: $news-main-1;
+}
+
+.email-signup-main:before {
+    height: 26px;
+    content: '';
+    background-color: $news-main-1;
+    display: block;
+}
+
+.email-signup {
+    background-color: $news-main-1;
+    min-width: 16em;
+}
+
+.content {
+    padding-bottom: $gs-baseline * 2;
+}
+
+.email-signup {
+    //No script tag
+    a {
+        color: #ffffff;
+        padding: 0 .3em;
```

Comment:
any reason for `em` here and elsewhere? generally in dotcom I think the convention is pixels everywhere. there's a CSS postprocessor to turn them all into `rem`